crates/bevy_pbr/src/render/pbr_lighting.wgsl

```diff
@@ -40,8 +40,57 @@
 //
 // The above integration needs to be approximated.
 
-// Calculations for both diffuse and specular lights.
+// Input to a lighting function for a single layer (either the base layer or the
+// clearcoat layer).
+struct LayerLightingInput {
```

Comment:
we might benefit from reordering fields here - i think the f32 ahead of the 16b aligned vec3s wastes some space. these are not i/o structs so i don't know how important it is.

Reply:
Reordered.